src/lua/init.c

```diff
@@ -438,6 +439,18 @@ static const char *lua_modules_preload[] = {
 	NULL
 };
 
+static const char * const lua_modules_post_box[] = {
```

Comment:
Creating new `static const char *val[]` here fails on checkpatch with `ERROR:STATIC_CONST_CHAR_ARRAY: static const char * array should probably be static const char * const`, so several minor thing were changed here.